### PR TITLE
Pulled out top nav items into list

### PR DESCRIPTION
**Pull request by helgridly**

### CODE REVIEW

File: src/cljs/org/broadinstitute/firecloud_ui/main.cljs [hunk 1]
```diff
@@ -74,7 +82,7 @@
    (fn [{:keys [props state]}]
      (let [nav-context (nav/parse-segment (:nav-context props))
            page (keyword (:segment nav-context))]
-       (when-not (contains? #{:import-data :workspaces :methods} page)
+       (when-not (some #{page} (map :key top-nav-bar-items))
```
Comment (helgridly) [inline src/cljs/org/broadinstitute/firecloud_ui/main.cljs]: Alternatively: `when-not (contains? (set (map :key top-nav-bar-items)) page)`

File: src/cljs/org/broadinstitute/firecloud_ui/main.cljs [hunk 2]
```diff
@@ -58,15 +72,9 @@
...
-                      :onClick (fn [e] ((:on-nav props) :import-data))}]
-      [TopNavBarLink {:name "Workspaces"
-                      :selected (= (:selected-item props) :workspaces)
-                      :onClick (fn [e] ((:on-nav props) :workspaces))}]
-      [TopNavBarLink {:name "Method Repository"
-                      :selected (= (:selected-item props) :methods)
-                      :onClick (fn [e] ((:on-nav props) :methods))}]])})
+      (map (fn [item] [TopNavBarLink {:name (item :name)
+                                      :selected (= (:selected-item props) (item :key))
+                                      :onClick (fn [e] ((:on-nav props) (item :key)))}]) top-nav-bar-items)])})
```
Comment (helgridly) [inline src/cljs/org/broadinstitute/firecloud_ui/main.cljs]: Is this alignment (over the `map`) okay?

File: src/cljs/org/broadinstitute/firecloud_ui/main.cljs (hunk 1 above)
Comment (dmohs) [inline src/cljs/org/broadinstitute/firecloud_ui/main.cljs]: I would vote for this. Clearer in my opinion.

Comment (helgridly) [inline src/cljs/org/broadinstitute/firecloud_ui/main.cljs]: "this" = the alternative?

File: src/cljs/org/broadinstitute/firecloud_ui/main.cljs (hunk 2 above)
Comment (dmohs) [inline src/cljs/org/broadinstitute/firecloud_ui/main.cljs]: Pull `top-nav-bar-items` to a new line so it doesn't get lost in the end. Otherwise good.

**Comment by dmohs** [comment]
Looks great.

File: src/cljs/org/broadinstitute/firecloud_ui/main.cljs (hunk 1 above)
Comment (dmohs) [inline src/cljs/org/broadinstitute/firecloud_ui/main.cljs]: `(contains? (set (map :key top-nav-bar-items)) page)`

**Pull request merged**